### ISSUE TITLE
[REQUEST]: jump to a folder?

### ISSUE THREAD

**Issue by msapplewatch** [Vinzent03/obsidian-advanced-uri#113]
I was wondering if this would be possible to ask for a advanced-uri command to jump to and focus an obsidian folder (rather then just a fie)?

this could be very useful for vault navigation and other purposes

thx so much for your consideration

Z

**Comment by Vinzent03** [comment]
How do you open a folder in Obsidian? I don't know what you mean?

**Comment by msapplewatch** [comment]
thx!

I meant file folders where markdown files are stored (ie folders in the vault).

for example see image:

![CleanShot 2023-02-16 at 18 12 44](https://user-images.githubusercontent.com/75834902/219423556-13141a2c-e00e-4563-8506-85997aaef8d7.png)


so instead of a URI command to open and shoe a  file.md I want to open .obsidian and show/focus on a specific value folder

does this make sense?

thx!!

**Comment by Vinzent03** [comment]
I'm still confused, are you talking about a community plugin?

**Comment by msapplewatch** [comment]
Hi again

sorry no just normal obsidian or system folder. let me try to explain again.

currently advanced-uri can open and file and focus it ie:

```
obsidian://advanced-uri?vault=PKM&filepath=100-199%20Food/101%20Lib/%SystemClipboard%&viewmode=source&data=%OBS_data%&line=3
```

what I want is not to open and focus a note (.md file) but open and focus a folder (in the obsidian file browser):

![CleanShot 2023-02-17 at 10 00 37](https://user-images.githubusercontent.com/75834902/219586081-1608b9b5-c121-4d4e-b86b-60787330a14f.png)


so in this case I launch the advanced-uri and it focuses the obsidian UI on that folder (inbox in the above example)

does that make more sense 😊?

**Comment by Vinzent03** [comment]
So you don't want to open anything in a tab, but just focus a folder in the file explorer?

**Comment by msapplewatch** [comment]
yes, to see what in that folder, thank you again for your time and consideration

**Comment by zeltak** [comment]
Just saw that now you can bookmark folders and when executed they go back and highlight the folder in the file browser! maybe supporting bookmarks in `obsidian-advanced-uri` could be the solution?

**Comment by Vinzent03** [comment]
@zeltak  Thanks for the tip. I've found now a way to focus on a item in the file explorere, but I'm unsure if that feature is really a new parameter worth. Instead using the bookmarks plugin might be the better solution. So that you can open your bookmark via uri by specifying the title you've set.

**Comment by zeltak** [comment]
yeah that could be great!

Z

**Comment by Vinzent03** [comment]
Available in new release via the `bookmark` parameter. Read the docs [here](https://vinzent03.github.io/obsidian-advanced-uri/actions/bookmarks)

**Comment by zeltak** [comment]
thx so much @Vinzent03 !

I just tried it but does not seem to work for me? im sure im doing something wrong on my side

I tried this:

obsidian://advanced-uri?vault=iPKM&bookmark=panel&openmode=tab

where my vault is `iPKM` and the bookmark is called `panel`

is this correct syntax?

**Comment by Vinzent03** [comment]
Works for me, what exactly did you bookmark?

**Comment by zeltak** [comment]
well I tried with normal markdown, canvas, folder etc

none (to me) seems to work. any tips on how to debug this?

thx a lot!

Z
